include/openenclave/bits/sgx/seal.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_BITS_SGX_SEAL_H
+#define _OE_BITS_SGX_SEAL_H
+
+#include <openenclave/bits/sgx/sgxtypes.h>
+
+#define OE_SEAL_SGX_SPECIFIC    1ULL
+
+typedef struct _oe_sealed_blob_header
+{
+    sgx_key_request_t   key_info;
+    uint32_t            aad_offset;     /* byte offset of the Additional
+                                           Authenticated Data in the sealed
+                                           blob */
+    uint8_t             reserved1[12];
```

Comment:
Can you comment on why these fields are there? Are they for some kind of alignment or future use purpose?

Reply:
I can add comments to them. They are for compatibility/interoperability with Intel SGX SDK.